build.yaml

```diff
@@ -69,6 +71,40 @@ release:
             ./build_${PACKAGE_TYPE}.sh
             find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \;
             pushd packages
+
+            # Install cpp-driver packages
+            sudo ${PACKAGE_INSTALL} cassandra*.${PACKAGE_TYPE}
+
+            # Create test file against cpp-driver and try to connect. Should compile and throw a connection error when executed
+            $CC -x c -o $TEST_PROGRAM - -lcassandra - <<EOF
+            #include <cassandra.h>
+            #include <stdio.h>
+
+            int main(int argc, char* argv[]) {
+            /* Setup and connect to cluster */
```

Comment:
These comment in the code are unnecessary for this.

Reply:
removed